aptos-move/framework/aptos-stdlib/sources/data_structures/big_vector.spec.move

```diff
@@ -65,6 +69,12 @@ spec aptos_std::big_vector {
         ensures result == spec_at(v, i);
     }
 
+    spec append<T: store>(lhs: &mut BigVector<T>, other: BigVector<T>) {
+        pragma verify=false;
+        // ensures forall i in 0..old(lhs).end_index: spec_at(old(lhs), i) == spec_at(lhs, i);
```

Comment:
Comments are left throughout the file in order to note the formulated specifications. Proving each of these specifications individually would be the next step.

Reply:
Maybe you could add another comment explaining that it is not proved but a work to be done next?